test/LowerColorado_TX/test_AnA.yaml

```diff
@@ -29,7 +29,7 @@ compute_parameters:
     compute_kernel         : V02-structured
     assume_short_ts        : True
     subnetwork_target_size : 10000
-    cpu_pool               : 36
+    cpu_pool               : 1

```

Comment:
Why are we changing this?

Reply:
just for test. Will be back to the original value.